src/esn.calendar.libs/app/core/auto-size-and-update.directive.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/**
+ * This directive enhances the auto-size directive of material admin.
+ * In fact, it corrects the initial height (i.e., when loading) of an autoSize element
+ * have a see: https://github.com/jackmoore/autosize/issues/248
+ */
+angular.module('esn.calendar.libs')
+  .directive('calAutoSizeAndUpdate', calAutoSizeAndUpdate);
+
+function calAutoSizeAndUpdate($timeout, autosize) {
+  var directive = {
+    restrict: 'A',
+    scope: true,
+    link: link
+  };
+
+  return directive;
+
+  ////////////
```

Comment:
What is the purpose of this comment?

Reply:
Just moving the file, not change the content.

BTW, the comment to separate the main part and the implementation part, IMO